src/components/Registration/Registration.component.js

```diff
@@ -197,29 +207,31 @@ const Registration = () => {
             name="instagram"
             value={contactInfo.instagram}
             onChange={handleChange}
-          />
+          />{" "}
           <div className="registerButtonContainer">
             <Button
               kind="primary"
               tabIndex={0}
               type="submit"
-              className="submitButton"
+              className="button primary"
             >
-              Register
-            </Button>
-          </div>
-        </Stack>
-      </Form>
+              Register{" "}
+            </Button>{" "}
+          </div>{" "}
+        </Stack>{" "}
+      </Form>{" "}
       {registrationNotif.isEnabled ? (
         <InlineNotification
-          style={{ marginTop: "2em" }}
+          style={{
+            marginTop: "2em",
+          }}
           kind="info"
           subtitle={`This is your member ID ${registrationNotif.id} Make sure your have this copied.`}
           title="You are now a member of Engineering 4.0!"
         />
       ) : (
-        <div></div>
-      )}
+        <div> </div>
```

Comment:
Just use <> </> Or try to avoid  (? :).  I think this can be done with (&&)

Reply:
replaced with &&